clients/venice-client/src/main/java/com/linkedin/venice/fastclient/DispatchingAvroGenericStoreClient.java

```diff
@@ -219,7 +244,8 @@ protected CompletableFuture<V> get(GetRequestContext requestContext, K key) thro
       }
     }
     if (transportFutures.isEmpty()) {
-      // No request has been sent out.
+      // No request has been sent out: If the first get() for the first route fails
+      // TODO: change the exception as there was a replica, but it failed to get data from that replica?
```

Comment:
Do we need this level of distinction?

Reply:
we do log the exceptions leading to this errors, so we should be okay. Resolving this.